langchain4j-cohere/src/main/java/dev/langchain4j/model/cohere/internal/api/ChatHistory.java

```diff
@@ -0,0 +1,15 @@
+package dev.langchain4j.model.cohere.internal.api;
+
+import lombok.Builder;
+import lombok.NonNull;
+
+@Builder
+public class ChatHistory {
```

Comment:
nit: `ChatMessage` or `Message` seems to be a better name.
History is the complete list of messages, not a single one.